APP-MANAGER

```diff
@@ -151,12 +155,43 @@ else
 	_appman
 fi
 
+# DOWNLOAD PORTABLE DEPENDENCES
```

Comment:
why not try installing jq through (apt, dnf, pacman, zipper) if failed then download through Am. so dostro may cause issue if you run rebuild package

Reply:
how should I do?

Reply:
check if apt, dnf, pacman or  zypper is present and if one is.. use that to install jq... 
(E.g. pacman -S jq)

Reply:
Wouldn't it be better to give the user this option to install it (so that they are free to use the package manager or AM)?

Reply:
> Wouldn't it be better to give the user this option to install it (so that they are free to use the package manager or AM)?

as it is a hard dependency for AM to work properly, i think its ok to make sure it is installed properly. without it your variables (E.g: version) my resolved to nothing and AM has many scary "rm -rf" spread across various scripts

Reply:
maybe I should leave jq as a dependence, without any workaround like this. Should I revert this to main?